apps/site/lib/site/application.ex

```diff
@@ -17,6 +17,10 @@ defmodule Site.Application do
 
     children = [
       # Start the endpoint when the application starts
+      supervisor(ConCache, [
+        [ttl: :timer.seconds(60), ttl_check: :timer.seconds(15)],
```

Comment:
In the ConCache README, I see only references to a `global_ttl` option for configuring the cache at the supervisor level, but no plain `ttl` option. Where is this documented?

Also, if the goal is "not more than a minute out-of-date", I think we should reduce the TTL and/or the check interval, since this means an item could be kept in the cache for up to 75 seconds. The TTL check is apparently an optimized process; given the docs recommend 1 second as a good default, I think we can feel very safe setting it to e.g. 5 seconds.

Reply:
https://hexdocs.pm/con_cache/0.12.1/ConCache.html. Remember that we're using an old version of ConCache here, the name got changed at some point since.

"So why not update ConCache?" Every single `Repo` we have is built on it, and given that this feature has a history of exploding in our faces, I'm not inclined to undertake the upgrade now.

Reply:
Will reduce the check interval however.